frontend/src/components/widgets/DataFrame/DataFrame.tsx

```diff
@@ -425,53 +452,93 @@ function DataFrame({
         }
       }}
     >
-      <GlideDataEditor
-        ref={dataEditorRef}
-        columns={columns}
-        rows={numRows}
-        minColumnWidth={MIN_COLUMN_WIDTH}
-        maxColumnWidth={MAX_COLUMN_WIDTH}
-        rowHeight={ROW_HEIGHT}
-        headerHeight={ROW_HEIGHT}
-        getCellContent={getCellContent}
-        onColumnResize={onColumnResize}
-        // Freeze all index columns:
-        freezeColumns={numIndices}
-        smoothScrollX={true}
-        // Only activate smooth mode for vertical scrolling for large tables:
-        smoothScrollY={numRows < 100000}
-        // Show borders between cells:
-        verticalBorder={true}
-        // Activate copy to clipboard functionality:
-        getCellsForSelection={true}
-        // Deactivate row markers and numbers:
-        rowMarkers={"none"}
-        // Deactivate selections:
-        rangeSelect={"rect"}
-        columnSelect={"none"}
-        rowSelect={"none"}
-        // Activate search:
-        keybindings={{ search: true }}
-        // Header click is used for column sorting:
-        onHeaderClicked={onHeaderClick}
-        gridSelection={gridSelection}
-        onGridSelectionChange={(newSelection: GridSelection) => {
-          setGridSelection(newSelection)
+      <Resizable
+        data-testid="stDataFrameResizable"
+        ref={resizableRef}
+        defaultSize={resizableSize}
+        style={{
+          border: `1px solid ${theme.colors.fadedText05}`,
         }}
-        theme={createDataFrameTheme(theme)}
-        onMouseMove={(args: GridMouseEventArgs) => {
-          // Determine if the dataframe is focused or not
-          if (args.kind === "out-of-bounds" && isFocused) {
-            setIsFocused(false)
-          } else if (args.kind !== "out-of-bounds" && !isFocused) {
-            setIsFocused(true)
-          }
+        minHeight={MIN_TABLE_HEIGHT}
+        maxHeight={maxHeight}
+        minWidth={MIN_TABLE_WIDTH}
+        maxWidth={maxWidth}
+        enable={{
+          top: false,
+          right: false,
+          bottom: false,
```

Comment:
we don't want this to be true right @LukasMasuch ?

Reply:
Nope, `bottom` and `right` is unfortunately overlaying the scrollbars.